### PR TITLE
Add DaylightingIlluminanceSetpointUnits element

### LINKED ISSUE #153: Daylighting setpoint needs to be footcandles (IP units)
Right now the daylighting setpoint annotation states lux. This should not cause backward compatibility issues unless users were reporting in lux.

### PR DESCRIPTION
**Pull request by markborkum**
See `proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md` for details.

Fixes #153

### CODE REVIEW

File: proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Justification
+
+Currently, the units of measure for the `auc:DaylightingIlluminanceSetpoint` element is given in lux.
+Other units of measure are defined in the literature, e.g., foot-candle [1].
+
+## Implementation
+
+This proposal is to add a new `auc:DaylightingIlluminanceSetpointUnits` element as an enumeration with the following values:
+* "fc" (Foot-candle)
+* "lux" (Lumens per square meter)
```
Comment (nllong) [inline proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md]: In general BSync does not use SI units at all, so I think we want just be explicit in the annotation that the units are foot-candles. Thoughts @markborkum?

File: examples/Reference Building - Primary School.xml [hunk 2]
```diff
@@ -180,6 +180,7 @@
                           </OccupancyScheduleIDs>
                           <OccupantsActivityLevel>Low</OccupantsActivityLevel>
                           <DaylightingIlluminanceSetpoint>500</DaylightingIlluminanceSetpoint>
```
Comment (nllong) [inline examples/Reference Building - Primary School.xml]: This value is clearly in Lux and should be 50 ft at most (https://www.lightingdesignlab.com/sites/default/files/pdf/Footcandle_Lighting%20Guide_Rev.072013.pdf)

File: proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md (hunk 1 above)
Comment (markborkum) [reply proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md]: @nllong This is definitely more straightforward than adding a new enumeration. Not very inclusive for users outside of USA, Liberia and Myanmar, on the other hand.

File: examples/Reference Building - Primary School.xml (hunk 2 above)
Comment (markborkum) [reply examples/Reference Building - Primary School.xml]: @nllong I added the "units" element. The value was already present in the example file. Should we change it?

File: proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md (hunk 1 above)
Comment (nllong) [reply proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md]: agreed that this is not friendly to most of the world. But it is what it is. We can create BSync-SI.xsd someday.

File: examples/Reference Building - Primary School.xml (hunk 2 above)
Comment (nllong) [reply examples/Reference Building - Primary School.xml]: yeah, we'll need to update the example files. Basically divide the values > 100 by 10 (10.76 to be precise).

**Comment by markborkum** [comment]
See #169 for new PR.

**Pull request closed without merging**